lib/phoenix/logger.ex

```diff
@@ -123,13 +123,13 @@ defmodule Phoenix.Logger do
   @doc false
   def install do
     handlers = %{
-      [:phoenix, :endpoint, :start] => &__MODULE__.phoenix_endpoint_start/4,
-      [:phoenix, :endpoint, :stop] => &__MODULE__.phoenix_endpoint_stop/4,
-      [:phoenix, :router_dispatch, :start] => &__MODULE__.phoenix_router_dispatch_start/4,
-      [:phoenix, :error_rendered] => &__MODULE__.phoenix_error_rendered/4,
-      [:phoenix, :socket_connected] => &__MODULE__.phoenix_socket_connected/4,
-      [:phoenix, :channel_joined] => &__MODULE__.phoenix_channel_joined/4,
-      [:phoenix, :channel_handled_in] => &__MODULE__.phoenix_channel_handled_in/4
+      [:phoenix, :endpoint, :start] => &phoenix_endpoint_start/4,
```

Comment:
@marcandre I think it is the intention to be this way in order to avoid telemetry warning. Ref #4401